src/main/java/com/WeatherIT/WeatherITApplicationController.java

```diff
@@ -9,5 +9,10 @@
 	public String start() {
 		return "index";
 	}
+	
+	@RequestMapping(value = "/login")
+	public String available() {
+		return "login";
+	}
 
```

Comment:
Added appropriate routing code for when you implement your login page based off of your storyboard.